### PR TITLE
Prepareing release 6.5.4

### PR DESCRIPTION
**Pull request by philkra**
* Backtrace limiting for a transaction
* casting error fixed for php <= `7.1`
* explanation for monitoring a curl call via transaction #53 & #51

### CODE REVIEW

**Comment by philkra** [comment]
@dstepe can I have pair of 👀 on this PR please, thanks and I gave you collaborator permissions on the repo.

**Comment by dstepe** [comment]
@philkra Did you get those review comments? I forgot to actually submit the review at first and wanted to make sure you saw it now.

File: src/Events/EventBean.php [hunk 1]
```diff
@@ -170,34 +170,34 @@ public function getTimestamp() : string
...
-    final protected function getEnv() : object
+    final protected function getEnv() : array
     {
         $envMask = $this->contexts['env'];
         $env = empty($envMask)
             ? $_SERVER
             : array_intersect_key($_SERVER, array_flip($envMask));
 
-        return (object) $env;
+        return (object)$env;
```
[the remark this answers is not in the document]
Comment (philkra) [reply src/Events/EventBean.php]: Oh boy, I missed that! Thanks!

**Comment by philkra** [comment]
@dstepe , yeah, sorry for the very sporadic replies! I already planed in more repo TLC in the next weeks :)

File: src/Events/EventBean.php [hunk 2]
```diff
@@ -170,34 +170,34 @@ public function getTimestamp() : string
      *
      * @link http://php.net/manual/en/reserved.variables.server.php
      * @link https://github.com/philkra/elastic-apm-php-agent/issues/27
+     * @link https://github.com/philkra/elastic-apm-php-agent/issues/54
      *
-     * @return object
+     * @return array
      */
-    final protected function getEnv() : object
+    final protected function getEnv() : array
```
[the remark this answers is not in the document]
Comment (philkra) [reply src/Events/EventBean.php]: I would go with array as return type and then cast it to ensure the backwards comp

**Comment by dstepe** [comment]
@philkra No problem. I completely understand competing priorities. Anyone who takes the time to work on open source solutions deserves thanks and patience. :)

**Pull request merged**